pydas/build.sh

```diff
@@ -0,0 +1,6 @@
+CC=${PREFIX}/bin/gcc
+CXX=${PREFIX}/bin/g++
+F77=${PREFIX}/bin/gfortran
+wget -P daspk31 http://www.cs.ucsb.edu/~cse/Software/daspk31.tgz
+make
```

Comment:
I might be missing something about how `make` works, but I don't understand what this is supposed to do.  Don't you have to unpack the tarball before you can compile anything?